config/zones.json

```
@@ -1224,7 +1225,12 @@
       ]
     ],
     "capacity": {
-      "hydro": 92
+      "battery storage": 0,
+      "hydro": 92,
+      "nuclear": 0,
+      "solar": 0.0104,
+      "wind": 0.81,
+      "unknown": 23.1796
     }
```

```suggestion
      "wind": 0.81
    }
```

Suggest to remove the `unknown` as per comment above.